tracer/test/Datadog.Trace.Security.IntegrationTests/IAST/AspNetCore5IastTests.cs

```diff
@@ -333,6 +333,27 @@ await VerifyHelper.VerifySpans(spansFiltered, settings)
                             .DisableRequireUniquePrefix();
     }
 
+    [SkippableFact]
+    [Trait("RunOnWindows", "True")]
+    public async Task TestIastHeaderTaintingRequest()
+    {
+        var filename = IastEnabled ? "Iast.HeaderTainting.AspNetCore5.IastEnabled" : "Iast.HeaderTainting.AspNetCore5.IastDisabled";
+        if (RedactionEnabled is true) { filename += ".RedactionEnabled"; }
+        var url = "/Iast/ExecuteCommandFromHeader";
+        IncludeAllHttpSpans = true;
+        AddHeaders(new Dictionary<string, string>() { { "file", "file.txt" }, { "argumentLine", "arg1" } });
```

Comment:
nit: target type new makes things nice and terse :D 
```suggestion
        AddHeaders(new () { { "file", "file.txt" }, { "argumentLine", "arg1" } });
```

Reply:
Thanks!